ros2_java_desktop.repos

```diff
@@ -19,6 +19,10 @@ repositories:
     type: git
     url: https://github.com/ros2/unique_identifier_msgs
     version: master
+  ros2_java/mockito_vendor:
+    type: git
+    url: https://github.com/ros2-java/mockito_vendor.git
+    version: add_package
```

Comment:
This is just temporary, until the [upstream PR](https://github.com/ros2-java/mockito_vendor/pull/1) is merged.

Reply:
Resolved in 2e5ccb8